fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -1060,6 +1060,17 @@ public CommandProcessingResult makeLoanRepaymentWithChargeRefundChargeType(final
                 transactionAmount, paymentDetail, noteText, txnExternalId, isRecoveryRepayment, chargeRefundChargeType, isAccountTransfer,
                 holidayDetailDto, isHolidayValidationDone);
         loan = loanTransaction.getLoan();
+        final LoanStatus loanStatus = loan.getStatus();
+        final boolean isBankChannel = channelData.getName().equals("Bancos")
+                || channelData.getHash().equals("1ae8d4db830eed577c6023998337d0hags546f1a3ba08e5df1ef0d1673431a3");
```

Comment:
are we sure this will never change ?

Reply:
Bancos channel name will be maintained unchanged by the system administrator
We can't use the ID because it will be random in the different deployment environments